src/player.ts

```diff
@@ -181,4 +185,15 @@ export default class Player {
     }
     this.data.power.gaia = 0;
   }
-}
+
+  buildingValue( building: Building, planet: Planet ){ 
+    const addedBescods = this.faction === Faction.Bescods && this.data[Building.PlanetaryInstitute] === 1  && planet === Planet.Titanium ? 1 : 0;
+    //TODO value if TECH3
+    return stdBuildingValue(building);
+  }
+
+  maxLeech( possibleLeech: number ){ 
+    // considers real chargeable power and victory points
+    return Math.min( this.data.chargePower(possibleLeech, true), this.data.victoryPoints);
```

Comment:
Instead of changing `this.data.chargePower`, you can have the max amount leechable with `this.data.power.bowl1 * 2 + this.data.power.bowl2`.

Also, as said before, it should be `this.data.victoryPoints + 1`.

Reply:
maybe maxLeech is misleading, it should be maxLeechable (e.g you have only 1 in bowl2 and can leech 3, your maxLeechable is 1, and you pay 0 VPs).

Reply:
Anyhow, the the second part of the comment still holds true. It should be `this.data.victoryPoints + 1`, since you can still leech 1 with 0vp

Reply:
right!